testing/cli/test_cli_sct_analyze_lesion.py

```diff
@@ -41,12 +41,38 @@ def dummy_lesion(request, tmp_path):
 
 
 def compute_expected_measurements(dim, starting_coord=None, path_seg=None):
-    # Find the minimum SC area surrounding the lesion
     if path_seg:
+        # Find the minimum SC area surrounding the lesion
         data_seg = Image(path_seg).data
         min_area = min(np.sum(data_seg[:, n_slice, :])
                        for n_slice in range(starting_coord[1], starting_coord[1] + dim[1]))
 
+        # Find the minimum mid-sagittal tissue bridge width for each LR slice in the lesion
```

Comment:
I've added the bridge widths to the "dummy lesion measurements" test.

My goal with this was to re-implement the calculations, to ensure that I fully understanding what's being proposed here.